src/index.js.flow

```diff
@@ -140,8 +141,12 @@ declare type ExpectChain<T> = {
   // chai-enzyme
```

Comment:
why is chai-enzyme even here? goodeggs-test-helpers doesn't come with chai-enzyme... we just accepted this silliness due to flow limitations, right? @dannynelson

Reply:
Yeah, flow limitations. The flow-typed definition has all this crap but is incomplete.

I should probably upstream this stuff, but it does fix the immediate problem.